### DISCUSSION TITLE
Welcome to HarmonyInstallGMS Discussions!

### DISCUSSION THREAD

**Discussion by to-alan** [to-alan/HarmonyInstallGMS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by D1scowolf** [comment]
现在华为的备份与恢复功能强制更新了，没有办法从内部存储中恢复，请问该如何解决呢（nova11）

**Reply by to-alan** [reply]
博客里有降级教程

**Reply by D1scowolf** [reply]
降级之后依然会自动更新，而且断网无效，能不能考虑根据这个报错“Failed checkUninstalledSystemApp during installPackageLI: Package com.huawei.localBackup only restored to uninstalled apk”找到这个uninstalled apk的位置，把它彻底删掉，再重装备份？
